src/app/app-routing.module.ts

```diff
@@ -38,13 +38,16 @@ const routes: Routes =  [
     path: 'players', component: PlayersListComponent,
   },
   {
-    path: 'players/interactive', component: InteractivePlayerComponent,
+    path: 'players/interactive/:id', component: InteractivePlayerComponent,
   },
   {
     path: 'players/collection', component: CollectionPlayerComponent,
   },
   {
     path: 'players/:mimeType/:id', component: PlayersComponent,
+  },
+  {
+    path: 'players/interactive', component: InteractivePlayerComponent,
```

Comment:
Why are we adding both here for InteractivePlayerComponent?

Reply:
InteractivePlayerComponent is not part of comman player component. hence we are handling separately and added new route for this.

Reply:
path: 'players/interactive', component: InteractivePlayerComponent,

path: 'players/interactive/:id', component: InteractivePlayerComponent,

Both the paths launching the same Component.

Reply:
`path: 'players/interactive', component: InteractivePlayerComponent,` 
The above path is added for loading default content from the player list.

`path: 'players/interactive/:id', component: InteractivePlayerComponent,`
This route added for loading content from the content list page (Review and publish the content)